Games/types/Mafia/roles/cards/ChangeRandomAlignment.js

```diff
@@ -0,0 +1,29 @@
+const Card = require("../../Card");
+const Random = require("../../../../../lib/Random");
+
+module.exports = class ChangeRandomAlignment extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.methods.changeAlignment = function() {
+            let alignment = Random.randArrayVal(["Village", "Mafia", "Cult"])

```

Comment:
imo , should it check which alignments are currently in the game/ which alignments the game started with?

also on EM it would predictably oscillate between vil and maf, allowing politician to plan

Reply:
It could check the different alignments, even which independents are in play so that it has more variety (for example the Politician could rotate between Village, Serial Killer, Turkey, Alien, Mafia, etc...)

I do like how Vastined's code makes the Politician not oscillate back and forth, because the added randomness is a bit of an extra challenge and feels like the best of both worlds between Politician and Nomad (and I'm not a fan of Nomad). It also feels better because not every game will last to day 3 or 4 anyway

Reply:
nomad can choose who to side with, politician's randomness means it can give flipped to ANYTHING the phase the game ends, in essence politician doesnt even have to play

Reply:
im just going to test that it works, you're the patrol guy xD